ktor-client/ktor-client-features/ktor-client-json/ktor-client-json-tests/jvm/src/io/ktor/client/features/json/tests/JsonTest.kt

```diff
@@ -151,7 +151,7 @@ abstract class JsonTest : TestWithKtor() {
     @Test
     fun testCustomContentTypesMultiple() = testWithEngine(CIO) {
         configCustomContentTypeClient {
-            acceptContentTypes = listOf(ContentType.Application.Json, customContentType)
+            accept(ContentType.Application.Json, customContentType)
```

Comment:
So we accept `application/json` twice?

Reply:
Well, I have no idea, I didn't change the logic, just updated the syntax in tests.

Reply:
Fixed.